testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentOptions.java

```diff
@@ -48,6 +48,9 @@
     @Option(names = "--debug", description = "Open Java debug ports")
     public boolean debug;
 
+    @Option(names = "--debug-suspend-tests", description = "Wait for client to connect in debug mode. Product Tests process only.")
+    public boolean debugSuspend;
```

Comment:
Hmm I was thinking about it but i thought it would be better to have it next to each other.
But seems there are more votes on TestRunOptions. Will switch to it.